OpenGlass/GlassFramework.cpp

```diff
@@ -332,6 +339,13 @@ void GlassFramework::UpdateConfiguration(ConfigurationFramework::UpdateType type
 		g_roundRectRadius = static_cast<int>(ConfigurationFramework::DwmGetDwordFromHKCUAndHKLM(L"RoundRectRadius"));
 	}
 
+	GlassSharedData::g_ColorizationAfterglowBalance = ((float)ConfigurationFramework::DwmGetDwordFromHKCUAndHKLM(L"OG_ColorizationAfterglowBalance",43) / 100);
+	GlassSharedData::g_ColorizationBlurBalance = ((float)ConfigurationFramework::DwmGetDwordFromHKCUAndHKLM(L"OG_ColorizationBlurBalance",49) / 100);
```

Comment:
By doing so, there is currently no GUI to set this value like `GlassOpacity`, are you sure about this?